prescient.el

```diff
@@ -314,7 +319,10 @@ enclose literal substrings with capture groups."
                (regexp-quote
                 (char-to-string char))
                with-groups))
-            subquery ".*"))))
+            subquery ".*"))
+          (`literal+initialism ;; For backwards compatibility
+           (let ((prescient-filter-method '(literal initialism)))
+             (car (prescient-filter-regexps subquery with-groups))))))
```

Comment:
This works, but I think it would be much more elegant and straightforward to instead change the `if` statement below to instead do:

* if it's the symbol `literal+initialism`, then return the list `(literal initialism)`
* if it's a list, then use as is
* otherwise, wrap in a list

What do you think?

Reply:
Sounds like a good idea to me